cves/kernel/CVE-2021-4001.yml

```diff
@@ -203,8 +179,10 @@ autodiscoverable:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  note:
-  answer:
+  note: |
```

Comment:
Your explanation for why you don't think it's autodiscoverable could use more detail; the problem being one of elevated permissions does not preclude an automated tool from automatically figuring out how to gain those permissions. Do you believe the sequence of actions is not one an automated tool could make on its own?